src/golang/cmd/server/queries/postgres.go

```diff
@@ -83,6 +87,10 @@ func (r *postgresReaderImpl) GetLoadOperatorSpecByOrganization(
 	organizationId string,
 	db database.Database,
 ) ([]LoadOperatorSpecResponse, error) {
+	// Get the artifact id, artifact name, operator id, workflow name, workflow id,
+	// and operator spec of all load operators (to_id) and the artifact(s) going to
+	// that operator (from_id; the table/object being written/saved) in the workflows
```

Comment:
What does "the table/object being written/saved" referred to? Is that part of the operator spec being returned?